inc/thorvg.h

```diff
@@ -389,15 +389,16 @@ class TVG_API Paint
     /**
      * @brief Gets the axis-aligned bounding box of the paint object.
      *
-     * In case @p transform is @c true, all object's transformations are applied first, and then the bounding box is established. Otherwise, the bounding box is determined before any transformations.
-     *
      * @param[out] x The x-coordinate of the upper-left corner of the object.
      * @param[out] y The y-coordinate of the upper-left corner of the object.
      * @param[out] w The width of the object.
      * @param[out] h The height of the object.
-     * @param[in] transformed If @c true, the paint's transformations are taken into account, otherwise they aren't.
+     * @param[in] transformed If @c true, the paint's transformations are taken into account in the scene its belonged. Otherwise they aren't.
      *
+     * @note This is useful when you need to figure out the bounding box of the paint in the canvas space.
      * @note The bounding box doesn't indicate the actual drawing region. It's the smallest rectangle that encloses the object.
+     * @note The paint needs to be pushed into a canvas if @p transformed is true, Update the paint beforehand to get the correct values.
```

Comment:
`If @p transformed is @c true, the paint needs to be pushed into a canvas and updated before this api is called.`